src/components/structures/MatrixChat.js

```diff
@@ -249,6 +250,10 @@ module.exports = React.createClass({
                 register_hs_url: paramHs,
             });
         }
+
+        this._sessionStore = sessionStore;
```

Comment:
Actually, no this was a side-effect of MatrixChat being the thing originally managing mx_pass.